components/modal/NewBoard.vue

```diff
@@ -24,17 +24,17 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>. -->
       <main class="min-w-[32rem] max-w-3xl" @keypress.enter="createNewBoard()">
         <div class="flex flex-row items-start justify-between">
           <h1 class="pointer-events-auto pr-5 text-2xl font-bold">
-            Create a new board
+            {{ $t("modals.newBoard.title") }}
           </h1>
           <XMarkIcon
             class="text-accent-hover size-6 cursor-pointer"
             @click="closeModal()"
           />
         </div>
         <section id="inputs" class="mt-4 flex flex-col">
-          <label class="text-medium text-dim-1 mb-2 text-lg" for="boardName"
-            >Board Name</label
-          >
+          <label class="text-medium text-dim-1 mb-2 text-lg" for="boardName">{{
+            $t("modals.newBoard.name")
+          }}</label>
           <input
```

Comment:
Placeholder of the input needs a translation

Reply:
ok fixed now